### PR TITLE
Учебный проект: pimp my mage

### PR DESCRIPTION
**Pull request by akzhar**
---
:mortar_board: [Учебный проект: pimp my mage](https://up.htmlacademy.ru/javascript/17/user/490939/tasks/13)

:boom: https://htmlacademy-javascript.github.io/490939-code-and-magick-17/7/

### CODE REVIEW

File: js/form.js [hunk 1]
```diff
@@ -2,16 +2,36 @@
 
 (function () {
   var dependencies = {
-    setup: window.setup
+    setup: window.setup,
+    dialog: window.dialog,
+    backend: window.backend
+  };
+  var Code = {
```
Comment (rkhaslarov) [inline js/form.js]: ENUM naming is a bit different according to the academy code style =)

File: js/backend.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  function onError(msgText) {
+    var header = document.querySelector('header');
+    var div = document.createElement('div');
+    div.style.backgroundColor = 'red';
+    div.style.height = '25px';
+    div.style.margin = '-25px 0 25px 0';
+    div.style.color = 'black';
+    div.textContent = 'ERROR: ' + msgText;
+    header.insertBefore(div, header.firstChild);
+  }
```
Comment (rkhaslarov) [inline js/backend.js]: The same as in main project

Comment (akzhar) [reply js/backend.js]: Fixed

File: js/form.js (hunk 1 above)
Comment (akzhar) [reply js/form.js]: If found the same in the lecture pdf:
var Code = { 
  SUCCESS: 200, 
};

**Pull request merged**